python/pyspark/sql/connect/dataframe.py

```diff
@@ -1775,6 +1775,10 @@ def _to_table(self) -> Tuple["pa.Table", Optional[StructType]]:
         assert table is not None
         return (table, schema)
 
+    def toArrowTable(self) -> "pa.Table":
```

Comment:
Can we list this to `python/docs/source/reference/pyspark.sql/dataframe.rst`?

Reply:
Let's also add the abstract method at the parent dataframe `pyspark.sql.dataframe`

Reply:
> Let's also add the abstract method at the parent dataframe `pyspark.sql.datafram`

I don't see where to do this. Can you clarify please? Thank you

Reply:
Here https://github.com/apache/spark/blob/master/python/pyspark/sql/dataframe.py :-)

Reply:
I think you should probably rebase it against the master branch. That refactoring happened lately (by me .. )

Reply:
Thanks.  I think I added what's needed there. It's a placeholder definition, like what you have there for `toPandas`.  See my other comment below.